policies/testing.md

```diff
@@ -42,6 +42,50 @@ A test may be omitted where writing the test would result in disproportionately
 more effort than writing the code being tested. For example, difficult to
 reproduce error conditions.
 
+Triage Labels
+-------------
+
+Before approving a PR, the applicability of the testing policy to the PR must
+be assessed and an appropriate label applied to the PR. One of the following
+labels must be applied:
+
+- `tests: present`, to be added where suitable tests are included in the same PR.
+- `tests: needed`, where the testing policy requires tests but they are not
+  currently included in a PR.
+- `tests: exempted`, where tests are not included in a PR but are not required
+  by this policy (for example, because the PR concerns only documentation,
+  or because writing the test would result in disproportionately more effort;
+  see above).
+- `tests: deferred`, where the tests are required by the testing policy
+  but it is intended to add these tests in a subsequent PR.
+- `tests: added in subsequent PR`, where the tests were deferred to a subsequent
+  PR which has now been merged. This may be set on a given PR after it is
+  merged, replacing the `tests: deferred` label.
+
+The `tests: needed` label blocks merging of the PR and constitutes a hold.
+None of the other labels listed above block merging of a PR.
```

Comment:
I would actually make that one a hold label, like "hold: tests needed".  That would make it clearer, as we're now all used to the "hold" labels as that sort of show stopper.

Reply:
Agreed - lets make this a hold